frontend/src/lib/components/DataDisplay/CardDisplay.svelte

```diff
@@ -27,36 +27,34 @@ export let styleProps = {
 	imgClass="max-md:hidden object-scale-down"
 	href={data.button ? null : data.href}
 	class={data.button
-		? 'm-2 max-w-prose items-center  text-gray-700 dark:text-white'
+		? 'm-2 max-w-prose min-w-full items-center bg-[#a4d2d0] text-gray-700 dark:text-white'
```

Comment:
I think the CardDisplay and GalleryDisplay components are used in multiple places, not just on the front page (@MaHaWo?).
It might make more sense for you to directly use a single `<Card>` element on the frontpage which you can customize however you like, rather than modifying these components?

Reply:
yes, they are indeed, I would also suggest to use `Card` directly if you need only one of them. also, I would suggest using colors that are defined centrally in `tailwind-config.ts`, because a unified style will depend on the color hexcode being copied everywhere we need it, which is brittle

Reply:
Thanks for your help and suggestions! I'll use the Card element in those cases and add other colors to tailwind-config.ts